django_oapif/__init__.py

```diff
@@ -1,5 +1,15 @@
+from . import permissions
+from .api import OAPIF
```

Comment:
Shall we stick only to one type of imports? I see almost everywhere else we use absolute imports, which are slightly more preferrable.

Reply:
Sure, I think we can even remove the re-export of permissions from here.